### PR TITLE
Hides company links without urls

### PR DESCRIPTION
**Pull request by Moejoe90**
#related https://github.com/Neufund/platform-frontend/issues/1948

I found that this happens because there are no url's so now this hides the url's. The solution to change the data structure is a bit more complicated

### CODE REVIEW

File: app/components/eto/public-view/DocumentsWidget.tsx [hunk 1]
```diff
@@ -45,15 +45,17 @@ const DocumentsWidgetLayout: React.SFC<TDispatchProps & TExternalProps & CommonH
...
-                  name={link.title || ""}
-                  altIcon={<InlineIcon svgIcon={icon_link} />}
-                />
-              </Col>
-            ))}
+            companyMarketingLinks.map((link, i) => {
+              return link.url && link.url !== "" ? (
+                <Col sm="6" md="12" lg="6" key={i} className={styles.document}>
+                  <DocumentLink
+                    url={link.url || ""}
```
Comment (drakkein) [inline app/components/eto/public-view/DocumentsWidget.tsx]: We don't need optional empty string, cuz we are not rendering anyway when url is empty

File: app/components/eto/public-view/DocumentsWidget.tsx [hunk 2]
```diff
@@ -45,15 +45,17 @@ const DocumentsWidgetLayout: React.SFC<TDispatchProps & TExternalProps & CommonH
...
-                <DocumentLink
-                  url={link.url || ""}
-                  name={link.title || ""}
-                  altIcon={<InlineIcon svgIcon={icon_link} />}
-                />
-              </Col>
-            ))}
+            companyMarketingLinks.map((link, i) => {
+              return link.url && link.url !== "" ? (
+                <Col sm="6" md="12" lg="6" key={i} className={styles.document}>
```
Comment (drakkein) [inline app/components/eto/public-view/DocumentsWidget.tsx]: I think we should use something more unique than index for a key, maybe url

Comment (Moejoe90) [reply app/components/eto/public-view/DocumentsWidget.tsx]: hmmm for now I would keep index the url can be repeated so its not unique

File: app/components/eto/public-view/DocumentsWidget.tsx (hunk 1 above)
Comment (Moejoe90) [reply app/components/eto/public-view/DocumentsWidget.tsx]: While you have a point i don't want to do any refactoring and `DocumentLink` expects both types

**Pull request merged**